includes/functions.php

```diff
@@ -681,15 +711,15 @@ function tsml_geocode($address) {
 		}
 	}
 
-	//cache result
-	$addresses[$address] = $response;
-	update_option('tsml_addresses', $addresses);
-
-	//add a status and return it
-	$response['status'] = 'geocode';
 	return $response;
 }
 
+//function: Call the new api gateway to geocode the address
```

Comment:
With the API gateway, I've tried to have it accept `url_encoded` data, and so I'm experimenting with just changing the URI. But, if I can't get that to work, we should pursue this I think.